rclcpp_examples/test/test_executables_example.py.in

```diff
@@ -0,0 +1,132 @@
+from importlib.machinery import SourceFileLoader
+import io
+import os
+import re
+import signal
+from launch import LaunchDescriptor
+from launch.exit_handler import ignore_exit_handler
+from launch.launcher import DefaultLauncher
+from launch.output_handler import LineOutput
+
+
+class InMemoryHandler(LineOutput):
+    def __init__(self, name, ld, expected_lines, regex_match=False):
+        super(LineOutput, self).__init__()
+        self.name = name
+        self.ld = ld
+        self.expected_lines = expected_lines
+        self.killed = False
+        self.left_over_stdout = b''
+        self.left_over_stderr = b''
+        self.stdout_data = io.BytesIO()
+        self.stderr_data = io.BytesIO()
+        self.regex_match = regex_match
+        self.matched = False
+
+    def on_stdout_lines(self, lines):
+        if self.killed:
+            return
+
+        for line in lines.splitlines():
+            # Filter out stdout that comes from underlying DDS implementations
+            if line.startswith(b'pid'):
+                continue
+            if line.startswith(b'rc'):
+                continue
+            if line.startswith(b'RTI Data Distribution Service Evaluation ' +
+                               b'License issued to'):
+                continue
+            if line.startswith(b'Expires on'):
+                continue
```

Comment:
For shorter code:

    filter_lines_starting_with = [...]
    if any([line.startswith(prefix) for prefix in filter_lines_starting_with]):
        continue